cns/service/main.go

```diff
@@ -932,10 +935,28 @@ func InitializeCRDState(ctx context.Context, httpRestService cns.HTTPService, cn
 		return err
 	}
 
+	// the nodeScopedCache sets Selector options on the Manager cache which are used
+	// to perform *server-side* filtering of the cached objects. This is very important
+	// for high node/pod count clusters, as it keeps us from watching objects at the
+	// whole cluster scope when we are only interested in the Node's scope.
+	nodeScopedCache := cache.BuilderWithOptions(cache.Options{
+		SelectorsByObject: cache.SelectorsByObject{
+			&v1alpha.NodeNetworkConfig{}: {
+				Field: fields.SelectorFromSet(fields.Set{"metadata.name": nodeName}),
+			},
+			&corev1.Pod{}: {
```

Comment:
Can we remove pods for now since we aren't using it? We have some tight rbac rules in place so adding this would require rbac change as well, which I don't think we're quite ready to make.

Reply:
dropped pods and nodes, maybe they will reappear at a later date :)